### DISCUSSION TITLE
Suggestion - Adding "Want to play" checkmark on Hubs / Game Lists

### DISCUSSION THREAD

**Discussion by telepathos** [RetroAchievements/RAWeb#1872] [Ideas]
As the title suggests, now that we have the new "Want to Play" feature it would be a functional upgrade if we could add games to the list from the Game Series list of games, since there's a lot of free space horizontally, the same could be said about the game lists (like console) though not as much space there atm. Just an easier way to add several games to the list without having to enter each game's page and clicking the button there.

Did a quick mockup here of the way it could be done:
![hhh](https://github.com/RetroAchievements/RAWeb/assets/62508928/da461d0b-6858-455e-9942-5e489a6c4f82)

**Comment by wescopeland** [comment]
I agree this would be nice to have. Alongside other hubs improvements, we should finish breaking out hubs from game pages before continuing to enhance hubs. Right now, hubs are magical game pages, and for all new code we add to hubs, we make game pages more complex.

**Reply by telepathos** [reply]
Just leaving it out there. I leave the magic up to you guys. Just felt this would be nice since I was in the process of mass adding games to the list (a great feature you guys added btw).

**Comment by Jamiras** [comment]
#2908 added it to games lists, and #2923 added it to hub lists.

**Reply by telepathos** [reply]
🙌
